File: Server/Model/Model/DTOAngajat.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Model.Model
{
    [Serializable]public class DTOAngajat
    {
        
        public DTOAngajat(string user,string pass)
        {
            this.Username = user;
            this.Password = pass;
        }


        public string Username { set; get; }
        public string Password { set; get; }
    }
}

File: Lab nertwork java/Lab1/Model/src/main/java/models/DTOAngajat.java
package models;


import java.io.Serializable;

public class DTOAngajat implements Serializable,Comparable<DTOAngajat> {
    private String username;
    private String password;
    public DTOAngajat(String user,String pass){
        this.username=user;
        this.password=pass;
    }

    public String getUsername() {
        return username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    @Override
    public String toString() {
        return "DTOAngajat{" +
                "username='" + username + '\'' +
                ", password='" + password + '\'' +
                '}';
    }

    @Override
    public int compareTo(DTOAngajat o) {
        return this.username.compareTo(o.username);
    }

    @Override
    public int hashCode() {
        return super.hashCode();
    }
}